enginetest/queries/queries.go

```diff
@@ -8446,6 +8446,20 @@ where exists (
 order by x, y;`,
 		Expected: []sql.Row{},
 	},
+	{
+		Query: "select dayname(123), dayname('abc')",
```

Comment:
some optional things that come to mind to maybe reverse engineer what the fuzzer is doing, could maybe run it through `typestable`, `select dayname(id), dayname(i8),... dayname(s1), dayname(NULL) from typestable limit 1;`, or have a query for each in the other places expressions can be, ORDER BY, HAVING, GROUP BY, JOIN ON, ... . Could in theory do this for each function